src/meshapi/views/geography.py

```diff
@@ -0,0 +1,171 @@
+from typing import Dict
+
+from django.db.models import F, Q
+from django.db.models.functions import Greatest
+from django.http import HttpResponse
+from fastkml import Data, ExtendedData, geometry, kml, styles
+from fastkml.enums import AltitudeMode
+from pygeoif import LineString, Point
+from rest_framework import permissions, status
+from rest_framework.decorators import api_view, permission_classes
+
+from meshapi.models import Building, Install, Link
+
+KML_CONTENT_TYPE = "application/vnd.google-earth.kml+xml; charset=utf-8"
+DEFAULT_ALTITUDE = 5  # Meters (absolute)
+
+ACTIVE_COLOR = "#F00"
+INACTIVE_COLOR = "#CCC"
+
+CITY_FOLDER_MAP = {
+    "New York": "Manhattan",
+    "Brooklyn": "Brooklyn",
+    "Queens": "Queens",
+    "Bronx": "The Bronx",
+    "Staten Island": "Staten Island",
+    None: "Other",
+}
+
+
+@api_view(["GET"])
+@permission_classes([permissions.AllowAny])
+def map_kml(request):
+    kml_root = kml.KML()
+    ns = "{http://www.opengis.net/kml/2.2}"
```

Comment:
Just a thought, we should document all our external dependencies somewhere 🤔

Reply:
Sure, but this isn’t a dependency. This is just a namespace for the XML schema that KML uses. We don’t call this at runtime

Reply:
oh, lol